draft-bbs-signatures.md

```diff
@@ -739,6 +739,10 @@ The ZKP protocols use nonces which MUST be different in each context.
 
 BBS signatures can be implemented on any pairing-friendly curve. Using BLS12-381 the signature achieves close to 128-bit security, and is the recommended curve at this time.
 
+## Nonce selection
```

Comment:
I think somewhere we need to relate the concept of a nonce to a presentation message, is here the most appropriate place to do that?

Reply:
Given that we now no longer formally have the concept of a nonce in the draft itself

Reply:
> Just so I can clarify when you say a more opinionated signature scheme what do you mean exactly, one that features holder cryptographic binding or something else?

Yes, at a minimum. This could be achieved (I think) by making one of messages the "private key" and you shouldn't reveal it. I've seen in another thread someone proposing tying it to the BLS key (?), I need to look into the details of that...

Reply:
> I think somewhere we need to relate the concept of a nonce to a presentation message, is here the most appropriate place to do that?

Ah yes, didn't realized that changed already when I merged. I definitely need to update that section to reflect that.

Reply:
Updated